src/services/nnid/routes/settings.ts

```diff
@@ -44,4 +67,110 @@ router.get('/profile', async function (request: express.Request, response: expre
 	}
 });
 
+/**
+ * [GET]
+ * Description: Fetches the requested mii image from the CDN and send it to the client.
+ * This is required because of the strict domain whitelist in the account settings app
+ * on the Wii U.
+ */
+router.get('/mii/:pid/:face', async function (request: express.Request, response: express.Response): Promise<void> {
+	const miiImage: Buffer = await got(`https://pretendo-cdn.b-cdn.net/mii/${request.params.pid}/${request.params.face}.png`).buffer();
```

Comment:
Can we avoid having hardcoded domains?

Reply:
`config.cdn.base_url` exists for this, to be clear